apps/ramp-network/package.json

```diff
@@ -11,7 +11,10 @@
   "scripts": {
     "start": "react-app-rewired start",
     "build": "react-app-rewired build",
-    "test": "react-app-rewired test --passWithNoTests"
+    "test": "react-app-rewired test --passWithNoTests",
+    "deploy:s3": "bash ../../scripts/deploy_to_s3_bucket.sh",
+    "deploy:pr": "bash ../../scripts/deploy_pr.sh",
+    "deploy:prod-hook": "bash ../../scripts/prepare-production-deployment.sh"
```

Comment:
Not related to this file, but we are missing the "project.json" for Ramp now :wink:

Reply:
good spot!